From 26bd74255d5fdf7a9dd0fb9140e0f6c2b9be5ac6 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 6 Feb 2021 12:44:30 -0800
Subject: [PATCH] compiler-rt: Include stddef.h

size_t is use in the source needs relevant header to include the
definition, fixes build on musl

compiler-rt/lib/fuzzer/FuzzerInterceptors.cpp:82:61: error: unknown type name 'size_t'; did you mean 'std::size_t'?

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 compiler-rt/lib/fuzzer/FuzzerInterceptors.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/compiler-rt/lib/fuzzer/FuzzerInterceptors.cpp b/compiler-rt/lib/fuzzer/FuzzerInterceptors.cpp
index b87798603fda..d5b0a42fd3fb 100644
--- a/compiler-rt/lib/fuzzer/FuzzerInterceptors.cpp
+++ b/compiler-rt/lib/fuzzer/FuzzerInterceptors.cpp
@@ -25,6 +25,7 @@
   }
 
 #include <cassert>
+#include <cstddef> // for size_t
 #include <cstdint>
 #include <dlfcn.h> // for dlsym()
 
